src/__tests__/starWarsConnectionTests.js

```diff
@@ -9,7 +9,7 @@
 
 import { expect } from 'chai';
 import { describe, it } from 'mocha';
-import { StarWarsSchema } from './starWarsSchema.js';
+import { StarWarsSchema } from './starWarsSchema';
```

Comment:
@motiz88 If you have time maybe it worth to split such changes into separate PR.
I think it would be easier to find someone from Facebook willing to merge such simple change
+ it will make the main PR smaller